.storybook/tailwind.config.cjs

```diff
@@ -4,5 +4,10 @@ module.exports = {
   ...tailwindConfig,
   content: [
     './src/**/*.{ts,tsx}'
-  ]
+  ],
+  safelist: [
```

Comment:
This adds tailwind classes to storybook to be used outside of a story in the ComponentMeta setup region. This was needed for `h-screen` to make the map full width. I figured why not allow all classes since this only affects the storybook environment, but I can limit to just what I need as well.